tools/harness-automation/autothreadharness/settings_sample.py

```diff
@@ -59,16 +59,19 @@
 THREAD_SED_POLLING_INTERVAL = 0
 """int: SED polling interval in seconds. Set to 0 to use Harness's default value."""
 
-HARNESS_HOME = 'C:\\GRL\\Thread1.1'
+HARNESS_HOME = "C:\\GRL\\Thread1.1"
 r"""str: Harness installation path, e.g. ``C:\GRL\Thread1.1``"""
 
-HARNESS_URL = 'http://127.0.0.1:8000'
+HARNESS_URL = "http://127.0.0.1:8000"
 """str: Harness front-end url"""
 
-TESTER_NAME = 'Thread Open'
+HARNESS_VERSION = "R1.4.0"
```

Comment:
Test Harness version is included in the Test Harness installation directory. 

`C:\GRL\Thread1.1\info.ini`
> [Thread_Harness_Info]
> Version = Thread V1.1 Test Harness Release 1.4.0

How about updating the automation tool to automatically get the version info of the installed Test Harness?

Reply:
good idea, there is a "Mode = External" flag to define the release mode for internal or external. will add it